interfaces/cython/cantera/transport.pyx

```diff
@@ -1,6 +1,12 @@
 # This file is part of Cantera. See License.txt in the top-level directory or
```

Comment:
I'm a little fuzzy on this, but I think the `.pyx` file inherits this from the corresponding `.pxd` file, if one exists. Since there is now a `.pxd` file for most `.pyx` files, these directives are mostly only needed in the `.pxd` files.